code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -80,12 +79,10 @@
 				new /obj/item/clothing/under/rank/medical/purple(src)
 				new /obj/item/clothing/head/surgery/purple(src)
 		new /obj/item/clothing/under/rank/medical(src)
-		new /obj/item/clothing/under/rank/nurse(src)
 		new /obj/item/clothing/under/rank/orderly(src)
 		new /obj/item/clothing/suit/storage/toggle/labcoat(src)
 		new /obj/item/clothing/suit/storage/toggle/fr_jacket(src)
 		new /obj/item/clothing/shoes/white(src)
-//		new /obj/item/weapon/cartridge/medical(src)

```

Comment:
Why is the PDA cartridge gone ?

Reply:
It was uncommented before, I though that it is safe to delete it as well, if it is dead code?

Reply:
Thats fine then